Tests/DefaultXpringClientTest.swift

```diff
@@ -5,11 +5,43 @@ final class DefaultXpringClientTest: XCTestCase {
 
   // MARK: - Balance
 
-  func testGetBalance() {
-    // GIVEN a XpringClient.
-    let xpringClient = DefaultXpringClient()
+  func testGetBalanceWithSuccess() {
+    // GIVEN a Xpring client which will successfully return a balance from a mocked network call.
+    let xpringClient = DefaultXpringClient(networkClient: FakeNetworkClient.successfulFakeNetworkClient)
 
-    // WHEN the balance is retrieved THEN an error is thrown.
+    // WHEN the balance is requested.
+    guard let balance = try? xpringClient.getBalance(for: .destinationAddress) else {
```

Comment:
Haven't used swift before. What is this . operator here? Where does it pull these values from?